src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1954,6 +2009,7 @@ public void processStanza(Stanza stanza)
             if (mucUser.getStatus().contains(MUCUser.Status.create(104)))
             {
                 updateFromRoomConfiguration();
+                updateMeetingId();
```

Comment:
Why do we need this, the id never changes. 
And this will do a second query for the room configuration ... another iq and response.